deepreg/predict.py

```diff
@@ -321,11 +324,13 @@ def main(args=None):
         "--gpu",
         "-g",
         help="GPU index for training."
+        "-g for using GPU remotely"
         '-g "" for using CPU'
         '-g "0" for using GPU 0'
         '-g "0,1" for using GPU 0 and 1.',
         type=str,
-        required=True,
+        required=False,
+        default=False,
```

Comment:
this arg is of type str, default should not be boolean?

I wonder, if we do not make it required, and there's no default, what will be the value?

Reply:
If there is no default， it gives None.
I tried the default can be bool False，
as the type is str.

Reply:
Should I just remove the default value？
If None can gives the same effect as False.